### ISSUE TITLE
Sankey fillOpacity does not work

### ISSUE THREAD

**Issue by deltaGPhys** [google/google-visualization-issues#2469]
Including with just the fiddle in the docs, fillOpacity seems to have no effect whatsoever, in either nodes or links. 

[Fiddle](https://jsfiddle.net/31p9dgbm/)

**Comment by dlaliberte** [comment]
Yes, apparently this has been broken for a while.  Thanks for your report.
